geode-dunit/src/main/java/org/apache/geode/management/internal/cli/commands/RebalanceCommandDistributedTestBase.java

```diff
@@ -94,6 +98,40 @@ public void testRebalanceResultOutput() {
     rebalanceResult.hasRow(9).contains(CliStrings.REBALANCE__MSG__MEMBER_COUNT);
   }
 
+  @Test
+  public void testRebalanceResultOutputMemberCount() {
+    server3 = cluster.startServerVM(3, "localhost", locator.getPort());
+    server3.invoke(() -> {
+      Cache cache = ClusterStartupRule.getCache();
+      RegionFactory<Integer, Integer> dataRegionFactory =
+          cache.createRegionFactory(RegionShortcut.PARTITION);
+      Region region = dataRegionFactory.create("region-1");
+      for (int i = 0; i < 100; i++) {
+        region.put("key" + (i + 400), "value" + (i + 400));
+      }
+    });
+    // check if DistributedRegionMXBean is available so that command will not fail
+    locator.waitUntilRegionIsReadyOnExactlyThisManyServers("/region-1", 3);
+
+    Map<String, List<String>> listMembersResult = gfsh.executeAndAssertThat("list members")
+        .hasTableSection().getActual().getContent();
+    assertThat(listMembersResult.get("Name").size()).isEqualTo(4);
+
+    server3.forceDisconnect();
+
+    Map<String, List<String>> rebalanceResult = gfsh.executeAndAssertThat("rebalance")
+        .statusIsSuccess().hasTableSection().getActual().getContent();
+    
+    server3.waitTilFullyReconnected();
+
+    listMembersResult = gfsh.executeAndAssertThat("list members")
+        .hasTableSection().getActual().getContent();
+    assertThat(listMembersResult.get("Name").size()).isEqualTo(4);
+    assertThat(rebalanceResult.get("Rebalanced Stats").get(9))
+        .isEqualTo("Total number of members in system on which rebalance is executed");
```

Comment:
This string should ideally be replaced with a reference to the `REBALANCE__MSG__MEMBER_COUNT` constant.

Reply:
Done! :)